package.json

```diff
@@ -2,10 +2,16 @@
   "name": "kagi_browser_ext",
   "version": "1.0.0",
   "description": "A simple helper extension for setting Kagi as a default search engine, and automatically logging in to Kagi in incognito browsing windows",
-  "main": "index.js",
+  "main": "build.js",
   "scripts": {
+    "watch-firefox": "nodemon build.js firefox watch",
+    "watch-chrome": "nodemon build.js chrome watch",
+    "build": "npm run build-firefox && npm run build-chrome",
```

Comment:
I added this to be closer to standards, simplify the building, test in CI, and eventually even make it easier to publish from the press of a button.